src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -395,6 +395,15 @@ export const SavedSearch = PropTypes.shape({
   type: PropTypes.oneOf([SEARCH_TYPES.ISSUE, SEARCH_TYPES.EVENTS]),
 });
 
+export const Incident = PropTypes.shape({
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  status: PropTypes.string.isRequired,
+  query: PropTypes.string,
+  projects: PropTypes.arrayOf(PropTypes.number),
+  dateCreated: PropTypes.string.isRequired,
```

Comment:
Is it possible that the model creation time is different than the incident start time?

Reply:
I'm not sure if that will be the case yet, I think we can add an incident start time field to this type when we decide

Reply:
We probably need resolved at and some other as well

Reply:
Other tables use `dateAdded` should we be consistent?

Reply:
I think created_at is the automatically created django datetime field?

Reply:
Do we need two created fields then?

Reply:
I think I'm gonna remove this for now, we can decide later